Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md

```diff
@@ -0,0 +1,43 @@
+---
+versionFrom: 8.0.0
+---
+
+# Connect to Azure Storage Explorer to upload files manually
+
+:::note
+**Important**: All Umbraco 8 projects created after May 6th 2020 will have Azure Blob Storage by default. That means that this guide only applies to your project has been created after May 6th 2020.
+:::
+
+In case you want to manually upload files to the blob storage provided to your site, you can take advantage of the software "Microsoft Azure Storage Explorer".
+
+:::note
```

Comment:
Should we make this a warning instead? Or is that perhaps a bit too much?

```suggestion
:::warning
```

Reply:
I believe it's fine with a warning - that stands out, and also ensures that there are not two notes right after each other 😅